### PR TITLE
Update bigdl-submit script

### PR DESCRIPTION
**Pull request by gc-fu**
## Description
Update Nano bigdl-submit script, so that it will be compatible with pert PyTorch distributed training application.

### 1. Why the change?

Update Nano bigdl-submit script, so that it will be compatible with pert PyTorch distributed training application.

### 2. User API changes
The following arguments are added:
use_command: If set, the script will use the command line arguments as pod's entrypoint.  Otherwise, it will be treated as entrypoint's arguments.

sgx_enabled: If set, the corresponding sgx-related device-plugin arguments will be added into resources/limits

node_label: Choose which node to run with labels

pod_epc_memory: The epc memory allocated to the sgx application. 

### 3. How to test?
- [x] Manually test

Due to an error occurred when installing bigdl, the PR has not been tested yet.
Will do later.

```bash
Collecting pyspark==2.4.6
  Using cached pyspark-2.4.6.tar.gz (218.4 MB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-_wqz2mzx/pyspark/setup.py'"'"'; __file__='"'"'/tmp/pip-install-_wqz2mzx/pyspark/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-_wqz2mzx/pyspark/pip-egg-info
         cwd: /tmp/pip-install-_wqz2mzx/pyspark/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-_wqz2mzx/pyspark/setup.py", line 156, in <module>
        long_description = pypandoc.convert('README.md', 'rst')
    AttributeError: module 'pypandoc' has no attribute 'convert'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

### CODE REVIEW

File: python/nano/src/bigdl/nano/k8s/bigdl_submit.py [hunk 1]
```diff
@@ -77,6 +77,12 @@ def _get_args_parser() -> ArgumentParser:
         default="1G"
     )
 
+    parser.add_argument(
+        '--pod_epc_memory',
+        type=str,
+        default="34359738368"
+    )
```
Comment (yangw1234) [inline python/nano/src/bigdl/nano/k8s/bigdl_submit.py:84]: Add a help string?

**Comment by yangw1234** [comment]
LGTM except one minor comments

File: python/nano/src/bigdl/nano/k8s/bigdl_submit.py (hunk 1 above)
Comment (gc-fu) [reply python/nano/src/bigdl/nano/k8s/bigdl_submit.py:85]: Fixed

**Comment by gc-fu** [comment]
Have tested it on ICX cluster and fixed a minor error

**Pull request merged**